feflow/tests/test_hybrid_topology.py

```diff
@@ -0,0 +1,143 @@
+                       force.getCutoffDistance(), "Expected different cutoff distances between NB and custom NB forces."
+        # propagate the cutoffs
+        hybrid_factory._add_nonbonded_force_terms()
+        # Check now that cutoff match for all nonbonded forces (including custom)
+        for force in old_nonbonded_forces:
+            for custom_force in hybrid_custom_nonbonded_forces:
+                assert custom_force.getCutoffDistance() == \
+                       force.getCutoffDistance(), "Expected equal cutoff distances between NB and custom NB forces."
+
+    def test_hybrid_topology_benzene_phenol(self, benzene, toluene, mapping_benzene_toluene,
+                                            standard_system_generator):
+        """
+        Test the creation of a HybridTopologyFactory object from scratch from a benzene to toluene
+        transformation.
+
+        Tests that we can create a HTF from scratch and checks that the difference in the number of
+        atoms in the Hybrid topology initial and final states is the expected one.
+
+        Returns
+        -------
+        None
+        """
+        benzene_offmol = benzene.to_openff()
+        toluene_offmol = toluene.to_openff()
+        # Create Openmm topologies from openff topologies
+        off_top_benzene = benzene_offmol.to_topology()
+        off_top_phenol = toluene_offmol.to_topology()
+        # Create openmm topologies initial and final states
+        initial_top = off_top_benzene.to_openmm()
+        final_top = off_top_phenol.to_openmm()
+        # Create openmm systems with the small molecules
+        system_generator = standard_system_generator
+        initial_system = system_generator.create_system(initial_top, molecules=[benzene_offmol])
+        final_system = system_generator.create_system(final_top, molecules=[toluene_offmol])
+        initial_positions = benzene_offmol.conformers[-1].to_openmm()
+        final_positions = toluene_offmol.conformers[-1].to_openmm()
+        # mapping
+        mapping = mapping_benzene_toluene.componentA_to_componentB  # Initial to final map
+        initial_to_final_atom_map = mapping
+        initial_to_final_core_atom_map = mapping
+        # Instantiate HTF
+        htf = HybridTopologyFactory(
+            initial_system,
+            initial_positions,
+            initial_top,
+            final_system,
+            final_positions,
+            final_top,
+            initial_to_final_atom_map,
+            initial_to_final_core_atom_map
+        )
+
+        # Validate number of atoms in hybrid topology end systems
+        n_atoms_diff = benzene_offmol.n_atoms - toluene_offmol.n_atoms  # Initial - Final -- Sign/order matters
+        initial_htf_n_atoms = len(htf.initial_atom_indices)
+        final_htf_n_atoms = len(htf.final_atom_indices)
+        assert initial_htf_n_atoms - final_htf_n_atoms == n_atoms_diff, \
+            "Different number of atoms in HTF compared to original molecules."
```

Comment:
To save time, we can stuff in a very brief test for `omm_hybrid_topology` here - just check in the atoms/bonds like we do here: https://github.com/OpenFreeEnergy/openfe/blob/afdffb1c9fa77e5df3b08851e917e7f66ff97a13/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py#L215-L227

I can do that alongside the tip4p-ew test for virtual sites tomorrow.